mayastor/src/core/bdev.rs

```diff
@@ -171,9 +171,9 @@ impl Bdev {
                 }
             }
             spdk_sys::SPDK_BDEV_EVENT_RESIZE => {
-                info!("Received resize event for bdev {}", bdev.name())
+                warn!("Received resize event for bdev {}", bdev.name())
             }
-            spdk_sys::SPDK_BDEV_EVENT_MEDIA_MANAGEMENT => info!(
+            spdk_sys::SPDK_BDEV_EVENT_MEDIA_MANAGEMENT => warn!(
```

Comment:
Perhaps this log's level was also appropriate at `info`... Would operators need to be aware of management plane events in their honeycomb/datadog instances?